src/DefaultPluginStarter.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace StellarWP\Telemetry;
+
+class DefaultPluginStarter extends PluginStarter {
+
+	public function __construct( array $args = [] ) {
+		$args = wp_parse_args( $args, [
+			'activation_hook'      => new DefaultActivationHook(),
+			'optin_template'       => new DefaultOptinTemplate(),
+			'plugin_slug'          => 'stellarwp-telemetry-starter',
+			'plugin_version'       => '1.0.0',
+			'option_to_inherit'    => 'freemius_inherit',
+			'create_settings_page' => true,
```

Comment:
Are we creating a settings page in the library? Or should the modal be added to their current settings page for the plugin they are integrating it with? @ChrisMKindred

Reply:
Oh I see, you are parsing the args - so they can change the location. I understand now.

Reply:
Yes but this is still a great point, I'll give this a second look and think about how to make it easier to use. 

This will change anyway when I introduce the DI container, but the option/value pairs will still be the same unless we decide to change something like the `freemius` default.